core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -315,11 +337,29 @@ export class TranslationModalComponent {
       hasDuplicateDescriptions, hasUntranslatedElements);
   }
 
+  modifyContentToValidate(text: string): string {
+    const lineBreakLessText = text.replace(/(\r\n|\n|\r)/gm, '');
+
+    // These fake wrappers are added in order to wrap all the unwrapped texts
+    // to validate. See issue https://github.com/oppia/oppia/issues/13173
+    const fakeDiv = $('<div />', {html: lineBreakLessText});
+    fakeDiv.contents().filter(function() {
+      if (this.outerHTML === '<p>&nbsp;</p>') {
+        this.parentElement.removeChild(this);
+      }
+      return (this.nodeType === 3);
```

Comment:
What does 3 refer to? Can you add a comment or make a local named variable for the 3?

Reply:
Sorry! I added the const.